homeassistant/components/sma_manager/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""SMA Manager Initializer"""
+
+#  Copyright (c) 2023.
+#  All rights reserved to the creator of the following script/program/app, please do not
+#  use or distribute without prior authorization from the creator.
+#  Creator: Antonio Manuel Nunes Goncalves
+#  Email: amng835@gmail.com
+#  LinkedIn: https://www.linkedin.com/in/antonio-manuel-goncalves-983926142/
+#  Github: https://github.com/DEADSEC-SECURITY
+
+# Home Assistant Imports
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+# 3rd Party Imports
+from sma_manager_api import SMA
```

Comment:
You need to add this requirement to the manifest

Reply:
It is, the package name and the library name are different. That package comes from the lib sma-manager
![image](https://github.com/home-assistant/core/assets/45077772/c7cb15a6-aff4-4873-badc-e7ca1e082a69)